File: 100 Days Coding Part-1/3. Alphabet or Not/Alphabet.py
#                              Day 3 Coding Statement 

# Write a Python program to identify if the character is an alphabet or not.

# Description: 
#     Take an input character from user and check whether it is an alphabet or not.

# • Input : A
# → Output:  Alphabet

# • Input: 7 / @ # 
# → Output: Not an alphabet

# Method_1 (using if else condition)

# ch = input("Please Enter any Character: ")
# if((ch >= 'a' and ch <= 'z') or (ch >= 'A' and ch <= 'Z')):
#     print("The entered Character ", ch ," is an Alphabet.")
# else:
#     print("The entered Character ", ch ," is not an Alphabet.")

# Method_2 (using "isalpha()")

# ch = input("Please Enter any Character: ")
# if(ch.isalpha()):
#     print("The en3tered Character ", ch ," is an Alphabet.")
# else:
#     print("The entered Character ", ch ," is not an Alphabet.")

#  Surprise Code for you all

# write a Python program to check character is Alphabet, Digit or special Character.

ch = input("Please Enter any Character: ")

if((ch >= 'a' and ch <= 'z') or (ch >= 'A' and ch <= 'Z')):
    print("The entered Character ", ch ," is an Alphabet.")
    
elif(ch >= '0' and ch <= '9'):
    print("The entered Character ", ch ," is a Digit.")
    
else:
    print("The entered Character ", ch ," is a Special Character.")

File: 100 Days Coding Part-1/3. Alphabet or Not/Alphabet.java
//                              Day 3 Coding Statement 

// Write a Java program to identify if the character is an alphabet or not.

// Description: 
//     Take an input character from user and check whether it is an alphabet or not.

// • Input : A
// → Output:  Alphabet

// • Input: 7 / @ # 
// → Output: Not an alphabet

// Method_1 (using if else condition)

// import java.util.*;
// public class Alphabet {
//     public static void main(String[] args){
//         Scanner sc = new Scanner(System.in);
//         System.out.print("Enter any Character: ");
//         char ch = sc.next().charAt(0);

//         if((ch >= 'a' && ch <= 'z') || (ch >= 'A' && ch <= 'Z'))
//             System.out.println("The Entered Character "+ch+" is an Alphabet.");
        
//         else
//             System.out.println("The entered character "+ch+" is not an Alphabet.");
//     }
// }

// Method_2 (Here we will use "Character.isAlphabetic()" function)

import java.util.*;
public class Alphabet {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter any Character: ");
        char ch = sc.next().charAt(0);

        if(Character.isAlphabetic(ch))
            System.out.println(ch + " is an Aplhabet.");

        else
            System.out.println(ch+" is not an Alphabet.");
    }
}